motionplan/plan.go

```diff
@@ -396,8 +396,12 @@ func (p PlanState) Serialize() map[string]interface{} {
 	for fName, conf := range p.configuration {
 		confMap[fName] = referenceframe.InputsToFloats(conf)
 	}
-	m["poses"] = poseMap
-	m["configuration"] = confMap
+	if p.poses != nil {
+		m["poses"] = poseMap
+	}
+	if p.configuration != nil {
```

Comment:
ah reading the comments on the thread on slack now.
it's because they'd be added as an empty set of framesystem inputs.
very counter-intuitive --  nice find!